### PR TITLE
refactor: remove Aragon plugins from the OSx core repository 

### PR DESCRIPTION
**Pull request by jordaniza**
## Description

Removes plugins from OSx core package into separate plugin repositories:

- Token Voting -> aragon/token-voting-plugin
- Multisig -> aragon/multisig-plugin
- Admin -> aragon/admin-plugin

Deleted AddressList as it is unused.

The full list of changes is itemised in the changelog and has been discussed at length by the OS team.

Task ID: [OS-1307](https://aragonassociation.atlassian.net/browse/OS-1307)

## Type of change

See the framework lifecycle in `packages/contracts/docs/framework-lifecycle` to decide what kind of change this pull request is.

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [x] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have updated the `DEPLOYMENT_CHECKLIST` file in the root folder.
- [x] I have updated the `UPDATE_CHECKLIST` file in the root folder.
- [x] I have updated the Subgraph and added a QA URL to the description of this PR.

[OS-1307]: https://aragonassociation.atlassian.net/browse/OS-1307?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Rekard0 — DISMISSED** [review]

File: packages/contracts/deploy/new/40_finalize-management-dao/99_verify_step.ts [hunk 1]
```diff
@@ -56,7 +57,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   });
 
   await checkPermission(managementDaoContract, {
-    operation: Operation.Revoke,
+    operation: Operation.Grant,
```
Comment (Michael-A-Heuer) [inline packages/contracts/deploy/new/40_finalize-management-dao/99_verify_step.ts:60]: Can we double-check that this one is correct?

Comment (Michael-A-Heuer) [reply packages/contracts/deploy/new/40_finalize-management-dao/99_verify_step.ts:60]: Seems correct

File: packages/contracts/hardhat.config.ts [hunk 2]
```diff
@@ -42,20 +42,13 @@ for (const network of Object.keys(hardhatNetworks) as SupportedNetworks[]) {
...
-  };
   const testingFork: TestingFork = {
     network: '',
     osxVersion: '',
     activeContracts: {},
   };
-  hre.aragonPluginRepos = aragonPluginRepos;
   hre.aragonToVerifyContracts = [];
-  hre.managementDAOMultisigPluginAddress = '';
+  hre.managementDAOMultisigPluginAddress = ''; // TODO
```
Comment (Michael-A-Heuer) [inline packages/contracts/hardhat.config.ts:51]: Do we still need this? What's left to be done here?

Comment (Michael-A-Heuer) [inline packages/contracts/hardhat.config.ts:51]:
```suggestion
  hre.managementDAOMultisigPluginAddress = ''; // TODO
```
```suggestion
  hre.managementDAOMultisigPluginAddress = ''; // TODO This must be removed after the deploy script got refactored (see https://github.com/aragon/osx/pull/582)
```

Comment (Michael-A-Heuer) [reply packages/contracts/hardhat.config.ts:51]: Remember to take care of removing these vars in https://github.com/aragon/osx/pull/582

File: packages/contracts/src/package.json [hunk 3]
```diff
@@ -16,7 +16,7 @@
   },
   "homepage": "https://github.com/aragon/osx#readme",
   "dependencies": {
-    "@aragon/osx-commons-contracts": "1.4.0-alpha.1",
+    "@aragon/osx-commons-contracts": "1.4.0-alpha.3",
```
Comment (Michael-A-Heuer) [inline packages/contracts/src/package.json:19]: The latest version is `1.4.0-alpha.5`
https://github.com/aragon/osx-commons/releases/tag/1.4.0-alpha.5-contracts

File: packages/contracts/src/package.json [hunk 4]
```diff
@@ -16,7 +16,7 @@
...
     "@ensdomains/ens-contracts": "0.0.11",
     "@openzeppelin/contracts": "4.9.5",
     "@openzeppelin/contracts-upgradeable": "4.9.5"
```
Comment (Michael-A-Heuer) [inline packages/contracts/src/package.json:22]: OZ 4.9.6 exits

File: packages/contracts/src/package.json (hunk 3 above)
Comment (Michael-A-Heuer) [reply packages/contracts/src/package.json]: This is very strange. We corrected it in https://github.com/aragon/osx/pull/591

File: packages/contracts/src/package.json (hunk 4 above)
Comment (Michael-A-Heuer) [reply packages/contracts/src/package.json]: This is very strange. We corrected it in https://github.com/aragon/osx/pull/591

**Review by banasa44 — APPROVED** [review]

**Review by heueristik — APPROVED** [review]
LGTM. The `ethersproject` dependencies in `yarn.lock` should be revisited in the follow-up task [OS-1343](https://aragonassociation.atlassian.net/browse/OS-1343).

File: yarn.lock
[the remark this answers is not in the document]
Comment (jordaniza) [reply yarn.lock:1]: https://aragonassociation.atlassian.net/browse/OS-1343

**Pull request merged**